systemd/enftun@.service.in

```diff
@@ -4,7 +4,8 @@ Documentation=https://github.com/xaptum/enftun
 After=enftun-setup@%i.service
 BindsTo=enftun-setup@%i.service
 After=network-online.target
-Wants=network-online.target
+Wants=network-online.target 
+Requires=xap-usb-gadget.service
```

Comment:
Agreed, let's not add this to the main `enftun` repo. The router card buildroot can install a custom `enftun@.service.in` that includes this line, instead of the enftun default.